source/MaterialXView/Material.cpp

```diff
@@ -95,6 +95,7 @@ bool Material::generateShader(mx::GenContext& context)
     materialContext.getOptions().hwTransparency = _hasTransparency;
     materialContext.getOptions().hwShadowMap = materialContext.getOptions().hwShadowMap && !_hasTransparency;
 
+    _hwShader = nullptr;
```

Comment:
Assignment could be removed as it's assigned again on the next row.

Reply:
Maybe just add this as a code comment since it's not obvious. In case someone else is tempted to remove this later :)